db/schema.rb

```diff
@@ -67,6 +67,7 @@
   create_table "messages", force: :cascade do |t|
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string   "emailadd"
```

Comment:
This shouldn't be here. Can you `rails db:migrate` and commit the changes that happened to `schema.rb`?